garden.turtle.Jellybean.yml

```diff
@@ -0,0 +1,36 @@
+---
+app-id: garden.turtle.Jellybean
+runtime: org.gnome.Platform
+runtime-version: master
```

Comment:
```suggestion
runtime-version: "45"
```

There is no master branch of runtime on Flathub

Reply:
Oops, forgot I had that on. Changing now